.github/workflows/propagate-doc-upwards.yml

```diff
@@ -5,13 +5,13 @@ on:
   workflow_dispatch:
     inputs:
       dry-run:
-        description: 'If true, run the merge locally but do not push to remote'
-        required: false
-        default: 'false'
+        type: boolean
+        description: 'Dry run: if checked, run the merge locally but do not push to remote'
+        default: false
       slack-notifications:
-        description: 'If true, send slack notifications when errors occur'
-        required: false
-        default: 'false'
+        type: boolean
+        description: 'If checked, send slack notifications when errors occur'
+        default: true
```

Comment:
I think it is better to change this default value to 'true'.
BTW, how did it work in the cron, as we can see that notifications are correctly sent by default?

Reply:
The condition for the Slack notification in the workflow.
`if: failure() && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.slack-notifications == 'true') )`

So when the cron starts, the inputs.slack-notifications are ignored.

Reply:
About the default value, is set to false en purpose. The input has been created mainly to troubleshoot slack issue, but not to send a message when the workflow manually.

When someone is triggering the workflow manually, it is supposed to monitor the run by themself. There is no need to pollute other Engineers with a slack notification.